doc/design/logging.md

```diff
@@ -152,14 +152,14 @@ See also the code example to [create a custom logger](#creating-a-custom-logger)
 
 This is currently only partly implemented.
 
-Customization will be done via the `log/logger.hpp` file. This is currently in
-`iceoryx_hoofs/include/iceoryx_hoofs/log/logger.hpp`. To enable the customization
+Customization will be done via the `iox/logger.hpp` file. This is currently in
+`iceoryx_hoofs/reporting/include/iox/logger.hpp`. To enable the customization
```

Comment:
I have to think a bit about this. The user is not supposed to ever include this file but only `logging.hpp`. Therefore it might make sense to have a separate `log` namespace for the implementation.

Reply:
I added a `log` folder and moved all logging related files except `logging.hpp` into it.